src/main/drivers/vtx_soft_spi_rtc6705.c

```diff
@@ -123,7 +123,21 @@ void rtc6705_soft_spi_set_channel(uint16_t channel_freq)
 
 void rtc6705_soft_spi_set_rf_power(uint8_t reduce_power)
 {
-    rtc6705_write_register(7, (reduce_power ? (PA_CONTROL_DEFAULT | PD_Q5G_MASK) & (~(PA5G_PW_MASK | PA5G_BS_MASK)) : PA_CONTROL_DEFAULT));
+    switch(reduce_power)
+    {
+        case 0:// 1dBm
+            rtc6705_write_register(7, 0x006A);
```

Comment:
> Also note that register 7 is reset to default (0x04FBD) any time register 1 is changed.
This means that you will have to reapply the power setting every time you change frequencies to maintain the desired power level.

This is interesting.